clang/include/clang/Tooling/Refactoring/Rename/SymbolName.h

```diff
@@ -27,19 +30,22 @@ namespace tooling {
 /// //       ^~ string 0 ~~~~~         ^~ string 1 ~~~~~
 /// \endcode
 class SymbolName {
```

Comment:
The upstreaming commit will not resemble any apple/llvm-project commit anyway because in our fork, we need to adjust the uses of the Apple-only `OldSymbolName` at the same time. I’ll just untangle all of this while upstreaming. Should be easier than trying to find the boundaries right now.